t/mojo/reactor_ev.t

```diff
@@ -8,307 +8,328 @@ plan skip_all => 'EV 4.32+ required for this test!'                  unless eval
 use IO::Socket::INET;
 use Mojo::Util qw(steady_time);
 
-# Instantiation
-use_ok 'Mojo::Reactor::EV';
-my $reactor = Mojo::Reactor::EV->new;
-is ref $reactor,               'Mojo::Reactor::EV',   'right object';
-is ref Mojo::Reactor::EV->new, 'Mojo::Reactor::Poll', 'right object';
-undef $reactor;
-is ref Mojo::Reactor::EV->new, 'Mojo::Reactor::EV', 'right object';
-use_ok 'Mojo::IOLoop';
-$reactor = Mojo::IOLoop->singleton->reactor;
-is ref $reactor, 'Mojo::Reactor::EV', 'right object';
+my ($reactor, $reactor2, $listen, $readable, $writable);
+my ($client,  $port,     $server, $timer,    $recurring, $id);
+my ($single,  $pair,     $one,    $two,      $last);
```

Comment:
These re-reviews are getting a bit tedious... it seems you've made variables arbitrarily global now. At the very least `$id`, `$single` and `$last` are here for no reason.

Reply:
And i probably missed another one.